src/components/ContactLocations/styles.css

```diff
@@ -0,0 +1,29 @@
+.contact-location-icons {
```

Comment:
I would recommend you comply with our tailwind usage for background colors

Reply:
I guess filter is NOT available in tailwindcss.